vmdb/app/controllers/catalog_controller.rb

```diff
@@ -665,6 +669,12 @@ def process_sts(sts, task, display_name = nil)
 
   private      #######################
 
+  def class_service_template(prov_type)
+    prov_type.starts_with?('generic') ?
```

Comment:
@bzwei i am returning class name back for generic as well as when it is not generic, we can extend this method later for other class types if we add more types later.